packages/pds/src/api/com/atproto/server/activateAccount.ts

```diff
@@ -0,0 +1,31 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { assertValidDidDocumentForService } from './util'
+import { BlockMap, CidSet } from '@atproto/repo'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.server.activateAccount({
+    auth: ctx.authVerifier.accessNotAppPassword,
+    handler: async ({ auth }) => {
+      const requester = auth.credentials.did
+
+      await assertValidDidDocumentForService(ctx, requester)
+
+      await ctx.accountManager.activateAccount(requester)
+
+      const root = await ctx.actorStore.read(requester, (store) =>
+        store.repo.storage.getRootDetailed(),
+      )
+      const commitData = {
+        cid: root.cid,
+        rev: root.rev,
+        since: null,
+        prev: null,
+        newBlocks: new BlockMap(),
```

Comment:
Should the signed commit block end-up in here?

Reply:
umm yes good catch 👍